step-capstone/src/Components/FormPopover.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Popover} from '@material-ui/core';
+import ItemForm from './ItemForm'
+
+export default function FormPopver(props) {
```

Comment:
Nice, I love a good utility function. If you think you'll make similar decisions in a few places (eg `onClose` will probably always have the same behavior), consider reducing the amount of things in props and instead taking a small number of arguments for what you really care about.

Reply:
Could you elaborate on this? Do you mean condensing repeated pass downs into singular objects to reduce clutter, for example, putting onClose, addItem, and onEdit into a singular functions object? I still need onClose to be passed down so I can't necessarily just not pass it into props.

Reply:
What I was trying to say is that if you will always be passing the same value for some options in `props`, it might be easier to pass a few individual arguments rather than a large `props` structure with a lot of options. However, it seems like this file is not in the PR anymore so resolving.